### PR TITLE
documentation update

### PR DESCRIPTION
**Pull request by j-chad**
# Description
updates readme to be much more informative and allow developers to quickly get a sense of the project + setup and run

Related to #92 

## Screenshots
https://github.com/j-chad/platform/blob/doc/readme-update/README.md

## Type of change
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [x] This change ~requires~ a documentation update

# Checklist:
Leave blank if not applicable

I have completed these steps when making this pull request:
- [x] I have assigned my name to the issue
- [x] I have moved the issue to the **In Progress** column
- [x] I have labelled the PR appropriately
- [x] I have assigned myself to the PR

Before opening the PR for review:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have moved the linked issue to the **Review in Progress** column

### CODE REVIEW

**Review by wshe874 — DISMISSED** [review]
lgtm

**Comment by j-chad** [comment]
this is in draft still

**Comment by frasermcc9** [comment]
Could you add branch naming conventions and code review rules to `contributing.md`

**Comment by j-chad** [comment]
> Could you add branch naming conventions and code review rules to `contributing.md`

what conventions and rules are we adding?

**Review by frasermcc9 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,40 +1,106 @@
...
+```bash
+# Clone this repository
+$ git clone https://github.com/701-T4/flatshare
 
-## Attributing third party dependencies
+# Go into the repository
+$ cd flatshare
 
-All third party dependencies used in this project are attributed in the LICENSE-3RD-PARTY files. The files are generated automatically when there is an update to the dependecies used.
+# Install dependencies
```
Comment (alyu201) [inline README.md:52]: Do we have a preference or requirement to install frontend/backend dependencies only in their respective folders?

Comment (j-chad) [reply README.md:52]: that is the current implementation. If we change it later I will update the readme

**Review by oscarli00 — APPROVED** [review]

**Review by yzhe819 — APPROVED** [review]

**Pull request merged**